pkg/nodeagent/runtime/cri/interface.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cri
+
+import (
+	"centaurusinfra.io/fornax-serverless/pkg/nodeagent/runtime"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	criv1 "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+type RuntimeService interface {
```

Comment:
what is the relation of this interface to the CRI runtime service?

Reply:
this is actually cri runtime service interface.
old runtime interface in runtime/interface.go is repurposed to generalize as a interface of runtime dependency of node agent. including cadvisor, cri,  etc. in future to add cpu/memory resource admission and storage.